etc/tendrl/node-agent/node-agent-dev.conf.yaml

```diff
@@ -2,7 +2,6 @@
 etcd_port: 2379
 etcd_connection: 0.0.0.0
 log_cfg_path: /etc/tendrl/node-agent/node-agent_logging.yaml
-log_level: DEBUG
 package_source_type: pip
```

Comment:
why remove the above line? In dev env DEBUG is fine.

Reply:
No this is not used in anywhere, the log is completely  depending on sys log file

Reply:
Ack